docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+services:
+    frontend:
+        build:
+            context: ./frontend/
+            dockerfile: Dockerfile
+        image: frontend:deploy
+        container_name: frontend-deploy
+        ports:
+            - 81:81
```

Comment:
the same thing that zwzone mentioned here

Reply:
Will be done...